apps/rehabstod/src/pages/Patient/components/PatientRekoStatus.tsx

```diff
@@ -16,8 +12,6 @@ export function PatientRekoStatus({
   isDoctor: boolean
 }) {
   const { data: populatedFilters } = useGetSickLeavesFiltersQuery()
-  const { state } = useLocation()
```

Comment:
This state can be removed from `apps\rehabstod\src\pages\CurrentSickLeaves\components\TableBodyRows.tsx`as well.